language/move-command-line-common/Cargo.toml

```diff
@@ -12,10 +12,12 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0.52"
 difference = "2.0.0"
+dirs-next = "2.0.0"
```

Comment:
Nit. Is this a better (newer?) version of the `dirs` crate (https://crates.io/crates/dirs)?

Reply:
We chose version 2.0.0 because it already existed in [another place](https://github.com/move-language/move/blob/main/language/tools/move-package/Cargo.toml#L26).
We can update both later.